incubator/awx/templates/cache_deployment.yaml

```diff
@@ -30,16 +30,16 @@ spec:
         - name: {{ .Values.cacheName }}
           containerPort: {{ .Values.cacheService.port }}
         resources:
-{{ toYaml .Values.cacheResources | indent 10 }}
-    {{- with .Values.cacheNodeSelector }}
+          {{ toYaml .Values.cacheResources | indent 10 }}
```

Comment:
You should not indent `toYaml` statements. This won't work. Please revert these changes throughout the chart.

Reply:
I changed everything as you said.

Please review it again.